static/multifile-service.ts

```diff
@@ -133,7 +133,7 @@ export class MultifileService {
 
         const zip = await JSZip.loadAsync(f);
 
-        zip.forEach(async (relativePath, zipEntry) => {
+        await Promise.all(zip.map(async (relativePath, zipEntry) => {
```

Comment:
Perhaps we should turn this into a normal for loop, it doesn't make sense to use map here, as we never actual return anything. And in that case the return statements can be just continue's

Reply:
oh it needs both key and value, hmm

Reply:
nevermind then, as long as it works

Reply:
Up to you: this way the code runs concurrently (as all the promises run together, as they did in the old code). Making it a regular loop would run sequentially (which is cool too, just wasn't sure enough so replaced like with like).